allennlp/models/archival.py

```diff
@@ -55,7 +55,7 @@ def archive_model(serialization_dir: str,
         logger.error("weights file %s does not exist, unable to archive model", weights_file)
         return
 
-    config_file = os.path.join(serialization_dir, "model_params.json")
+    config_file = os.path.join(serialization_dir, "config.json")
```

Comment:
You could use the constant here, too, and in the other places you changed `"model_params.json"` to `"config.json"`.